os/arch/arm/src/amebasmart/amebasmart_pmhelpers.c

```diff
@@ -51,7 +51,7 @@
 #include "timer_api.h"
 
 gtimer_t g_timer1;
-extern struct timer_s g_timer_wakeup;
+extern struct pm_wakeup_timer_s g_timer_wakeup;
```

Comment:
Please separate this into a different commit as a fix

Reply:
This fix is already separated on an independent commit